toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -53,18 +53,6 @@ func (s *SystemConfig) GetRootPartitionSetting() (rootPartitionSetting *Partitio
 	return nil
```

Comment:
I changed the function to take a `[]PartitionSetting` instead of a `*SystemConfig`. I figured it made sense to move the location of the function since it was no longer related to `SystemConfig`.